packages/swirl-components/src/components/swirl-card/swirl-card.tsx

```diff
@@ -36,7 +46,16 @@ export class SwirlCard {
           }
           target={this.linkTarget}
         >
-          <slot></slot>
+          <div class="image-section">
+            <slot name="image"></slot>
+          </div>
+          <div class="heading-section">
+            <slot name="sub-heading"></slot>
```

Comment:
**suggestion:** Maybe a single `header` slot would be enough. You could put both the heading und sub heading inside.